src/lib/codec-importer.ts

```diff
@@ -0,0 +1,47 @@
+import multicodecs from 'multicodec'
+import { BlockCodec } from 'multiformats/codecs/interface'
+import type { PBNode } from '@ipld/dag-pb'
+import type { IPLDFormat } from 'blockcodec-to-ipld-format'
+
+export default async function codecImporter<T extends any = unknown>(codecCode: number): Promise<BlockCodec<any, any> | IPLDFormat<any>> {
+  // const codecName: string = multicodecs.codeToName[codecCode as CodecCode]
+  // console.log(`codecName: `, codecName);
+  switch (codecCode) {
+    case multicodecs.DAG_CBOR:
+      return await import('@ipld/dag-cbor') as BlockCodec<typeof multicodecs.DAG_CBOR, T>
```

Comment:
We need to add DAG_JSON too, for completeness.

Reply:
Do you know of any good dag-json examples I can include?

Reply:
no examples, and we probably should not include any on the page -- in nearly all cases we want people to use more compact dag-cbor everywhere. dag-json is effectively just JSON representation of dag-cbor. there may be use cases where it is more desired/efficient to store JSON at rest, but that will be and edge case.

for testing, you can create one with CLI (`ipfs dag put --input-codec dag-json --store-codec dag-json`)

Reply:
done